src/public/meeting.ts

```diff
@@ -143,4 +154,77 @@ export namespace meeting {
     const messageId = sendMessageRequestToParent('meeting.getAuthenticationTokenForAnonymousUser');
     GlobalVars.callbacks[messageId] = callback;
   }
+
+  /**
+   * Allows an app to get the state of the live stream in the current meeting
+   * @param callback Callback contains 2 parameters: error and liveStreamState.
+   * error can either contain an error of type SdkError, in case of an error, or null when get is successful
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function getLiveStreamState(
+    callback: (error: SdkError | null, liveStreamState: LiveStreamState | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[get live stream state] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.getLiveStreamState');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * Allows an app to request the live streaming be started at the given streaming url
+   * @param streamUrl the url to the stream resource
+   * @param streamKey the key to the stream resource
+   * @param callback Callback contains 2 parameters: error and liveStreamState.
+   * error can either contain an error of type SdkError, in case of an error, or null when operation is successful
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function requestStartLiveStreaming(
+    streamUrl: string,
+    streamKey: string,
```

Comment:
streamKey might be optional, in some cases where the URL itself might contain the required key. Please check with @sachingupta how we can make this as an optional parameter.

Reply:
I've moved these parameters around such that `callback` is the first parameter, and `streamUrl` and `streamKey` are after, with `streamKey` being optional.